_posts/2022-09-27-call-for-python-software-review-editors.md

```diff
@@ -0,0 +1,90 @@
+come in. 
+
+With that below you will find a few updates.
+
+## Welcome Editor in Chief David Nicholson 
+
+[David Nicholson](https://github.com/NickleDave), who has been working with pyOpenSci for the past few years as 
+an editor will take on the Editor in Chief role! [You can learn more about that 
+role here](https://www.pyopensci.org/contributing-guide/open-source-software-submissions/editor-in-chief-guide.html) in our peer review guide. David will:
+
+* Be the first person to reach out to anyone submitting a new review or a pre-submission. 
+* He will perform initial software checks to ensure your package is ready for  
+* He will also facilitate kicking off the peer review process by finding an editor for your package
+
+David has a background behavioral / neuro / cognitive science and is a ML guru. 
+Of course,  he's a Pythonista thru and thru. We are lucky to have David as a 
+part of our pyOpenSci team. 
+
+Stay tuned for a blog post from David introducing himself.
+
+## We are creating an editorial board!
+
+To support our open peer review, we want to bring on a few people (like yourself?!) 
+to serve as guest editors / editors. Here's how this works:
+
+Editors at pyOpenSci make a minimum of a 1-year commitment with a possibility
+of longer given mutual consent. We also 
+support guest editors in some instances. You will be responsible for handling 3-4 
+package reviews during that year. If you are new to our organization, we start 
+you as a guest editor. What this means is that you will review one package and 
+we will support you in that process to ensure you learn the ropes. Then if you 
+are happy and work well with our editorial team we will bring you on as a 
+"full editor".
+
+
+You can read more about the [roles and responsibilities of and editor here](https://www.pyopensci.org/contributing-guide/open-source-software-submissions/editors-guide.html).
+
+### What background do I need to be an editor for pyOpenSci?
+
+We welcome applications from potential editors with significant experience in 
+one or more of the following areas: 
+* open source software, 
+* open science
+* software engineering
+* peer-review
+
+Members of the JOSS editorial team have diverse backgrounds. We welcome 
+JOSS editors from academia, government, and industry. We especially welcome 
+applications from prospective editors who will contribute to the diversity 
+(ethnic, gender, disciplinary, and geographical) of our board. We also value 
+having a range of junior and senior editors.
+
+
+### How do I apply?
+
+To sign up to be considered as an editor for pyOpenSci, please fill out the 
+form below:
+
+[Click here to apply to be an editor <i class="fa fa-4 fa-arrow-circle-right" aria-hidden="true"></i>](https://forms.gle/YH4kTeDFoYjDtefh7){: .btn .btn--info .btn--large}
+
+## Questions?
+If you have any questions, please reach out to me at leah at pyopensci.org or on 
+twitter at @leahawasser.
```

Comment:
```suggestion
twitter at [@LeahAWasser](https://twitter.com/LeahAWasser)```

* twitter can be case-sensitive when you try to @ user names
* might as well link to your twitter profile while you're at it